src/moai_adk/templates/.claude/hooks/alfred/TROUBLESHOOTING.md

```
@@ -0,0 +1,471 @@
+=== Hook Test ===
+$(echo '{"cwd": "."}' | uv run .claude/hooks/alfred/alfred_hooks.py SessionStart 2>&1)
+
+=== Settings ===
+$(cat .claude/settings.json 2>&1 | grep -A 10 "SessionStart")
+EOF
+
+cat /tmp/hooks_diagnostic.txt
+```
+
+### Report Issues
+1. **GitHub Issues**: https://github.com/modu-ai/moai-adk/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)